crates/typst/src/foundations/datetime.rs

```diff
@@ -45,6 +45,23 @@ use crate::World;
 /// )
 /// ```
 ///
+/// # Datetime and [duration]($duration)
+///
+/// You can get a [duration]($duration) by subtracting two datetime.
+/// You can also get a new datetime via adding or subtracting a [duration]($duration)
+/// ```example
+/// #let datetime_duration = datetime(day: 1, month: 3, year: 2024) - datetime(day: 1, month: 1, year: 2024)
+/// #datetime_duration.hours()
+///
+/// #let two_days_duration = duration(days: 2)
+///
+/// #let two_days_ago = datetime(day: 1, month: 3, year: 2024) - two_days_duration
+/// #two_days_ago.display()
+///
+/// #let two_days_later = datetime(day: 1, month: 3, year: 2024) + two_days_duration
+/// #two_days_later.display()
+/// ```
```

Comment:
```suggestion
/// # Datetime and duration
/// You can get a [duration] by subtracting two datetimes:
/// ```example
/// #let first-of-march = datetime(day: 1, month: 3, year: 2024)
/// #let first-of-jan = datetime(day: 1, month: 1, year: 2024)
/// #let distance = first-of-march - first-of-jan
/// #distance.hours()
/// ```
///
/// You can also add/subtract a datetime and a duration to retrieve a new,
/// offset datetime:
/// ```example
/// #let date = datetime(day: 1, month: 3, year: 2024)
/// #let two-days = duration(days: 2)
/// #let two-days-earlier = date - two-days
/// #let two-days-later = date + two-days
///
/// #date.display() \
/// #two-days-earlier.display() \
/// #two-days-later.display()
/// ```
```
I suggest the following edit. In particular:
- Kebab-case
- Shorter lines
- Split up into two examples, each introduced by a separate sentence
- Prose wrapped at 80 columns

Reply:
changed